src/pages/guides/rpcs-and-das.md

```diff
@@ -0,0 +1,73 @@
+---
+title: RPCs and DAS
+metaTitle: RPCs and DAS on the Solana Blockchain
+description: Learn about RPCS on the Solana blockchain and how DAS by Metaplex aids in storing and reading data on Solana.
+# remember to update dates also in /components/guides/index.js
+created: '06-16-2024'
+updated: '06-21-2024'
+---
+
+## Roles of an RPC on the Solana Blockchain
+Remote Procedure Calls (RPCs) are a crucial part of the Solana blockchain infrastructure. They serve as the bridge between users (or applications) and blockchain, facilitating interactions and data retrieval.
```

Comment:
Is it worth here or somewhere else calling out the role of validators?  When I first started out I remember wondering who were validators vs. RPC providers.

Reply:
added a brief validator and staking page.